Makefile

```
@@ -91,14 +93,16 @@ C_DIR_TEST_BUILD = test/c/build
 # Whether this is target/debug or target/release depends on the flags in the
 # `cheader` recipe.  For now, they're just hardcoded.
 C_CARGO_TARGET_DIR = target/release
-C_LIB_CARGO_BASENAME=libqiskit_cext
 ifeq ($(OS), Windows_NT)
 	C_DYLIB_EXT=dll
+	C_LIB_CARGO_BASENAME=qiskit_cext
 else ifeq ($(shell uname), Darwin)
 	C_DYLIB_EXT=dylib
+	C_LIB_CARGO_BASENAME=libqiskit_cext
```

I think you can just go ahead and have this be
```suggestion
	C_LIB_CARGO_FILENAME=libqiskit_cext.dylib
```
(analogous for the others) since now we're just defining the value of two variables in the if-else and merge them below, but there's no reason really for this extra step 🙂